tests/conftest.py

```diff
@@ -185,8 +185,10 @@ def pytest_collection_modifyitems(session, items):
     # Add squad markers to each test item based on filepath
     for item in items:
         # check, if test already have squad marker manually assigned
-        if any(map(lambda x: "_squad" in x.name, item.iter_markers())):
-            continue
+        for marker in item.iter_markers():
+            if "_squad" in marker.name:
+                squad = marker.name.split("_")[0]
+                item.user_properties.append(("squad", squad))
```

Comment:
what if `_squad` not in the item's markers?

Reply:
If there is no `_squad` in the item's existing markers we move onto the code below that will add the marker and property based on the [constant map](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L1540-L1558) we have defined for squads.